File: Lista-15/python/10.py
class Calculadora:
    def somar(self, *args):
        return sum(args)

# Exemplo de uso
calc = Calculadora()
print(calc.somar(2, 3))         # Saída: 5
print(calc.somar(2, 3, 4))      # Saída: 9

File: Lista-15/java/10.java
class Calculadora {
    public int somar(int a, int b) {
        return a + b;
    }

    public int somar(int a, int b, int c) {
        return a + b + c;
    }

    public static void main(String[] args) {
        Calculadora calc = new Calculadora();
        System.out.println(calc.somar(2, 3));     // Saída: 5
        System.out.println(calc.somar(2, 3, 4));  // Saída: 9
    }
}
